### PR TITLE
prepared release of version 3.3.0

### PR DESCRIPTION
**Pull request by sanchariGr**
**Proposed changes**:
- Version 3.3.0 🎉

### CODE REVIEW

File: CHANGELOG.mdx [hunk 1]
```diff
@@ -16,6 +16,24 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
...
+  than loading all events first and pushing them to the broker afterwards.
+
+### Bugfixes
+- [#11575](https://github.com/rasahq/rasa/issues/11575): Fix `BlockingIOError` when running `rasa interactive`, after the upgrade of `prompt-toolkit` dependency.
+
+### Improved Documentation
+- [#11571](https://github.com/rasahq/rasa/issues/11571): Rasa SDK documentation lives now in Rasa Open Source documentation under the _Rasa SDK_ category.
+
+
 ## [3.3.0a1] - 2022-10-10
```
Comment (sanchariGr) [inline CHANGELOG.mdx:37]: @m-vdb @ancalita should the alpha release details be present in the changelog or does this need to be removed?

Comment (m-vdb) [reply CHANGELOG.mdx:37]: yeah feel free to remove

**Review by m-vdb — APPROVED** [review]

File: CHANGELOG.mdx (hunk 1 above)
Comment (ancalita) [reply CHANGELOG.mdx:37]: To double-check, shouldn't the changelog entries that were against the alpha release now appear against the new minor?

Comment (sanchariGr) [reply CHANGELOG.mdx]: @ancalita you're right there were some duplications, the 3.3.0a1 entries were again duplicated in 3.3.0. I've done the cleaning needed, can you please also review 🙏

**Review by ancalita — APPROVED** [review]

**Pull request merged**